OasisDEX/oasis-borrow
# Auto features dropdown refinement

# [Auto features dropdown refinement](https://app.shortcut.com/oazo-apps/story/5780/sidebar-dropdown-labels)

![image](https://user-images.githubusercontent.com/16230404/186146367-776bff13-1164-4d1a-9176-b918217d6fee.png)
![image](https://user-images.githubusercontent.com/16230404/186146425-c9a5f344-8b33-41e9-80a9-f1a38243b27f.png)
  
## Changes 👷‍♀️

- Updated labels according to shortcut story - active features in long label are prefixed with "Manage", inactive with "Setup",
- refined whole dropdown into single method that could be used in the future in single sidebar approach for automation features.
  
## How to test 🧪

Check out if all automation features sidebar dropdown are working correctly and labels are updated properly according to changes information.

Additional info; Stop-Loss dropdown should be created as single instance instead of twice, separately for adjust and remove, but it's not worth the time right now since it's going to be rebuild in single sidebar.
